diff/walking/differ.go

```diff
@@ -94,7 +94,7 @@ func (s *walkingDiff) Apply(ctx context.Context, desc ocispec.Descriptor, mounts
 	if err != nil {
 		return emptyDesc, errors.Wrap(err, "failed to create temporary directory")
 	}
-	defer os.RemoveAll(dir)
+	defer os.Remove(dir)
```

Comment:
https://github.com/crosbymichael/containerd/blob/a8b543f87d6ea36b49d2301dfaaac3aeda644edd/diff/walking/differ.go#L102

we should output unmount err and return error When unmount fail .

Reply:
I filed a pr #1877 , PTAL